calico_containers/pycalico/datastore_datatypes.py

```diff
@@ -204,7 +209,6 @@ def to_json(self):
         json_dict = {"state": self.state,
                      "name": self.name,
                      "mac": self.mac,
-                     "container:if_name": self.if_name,
```

Comment:
You're getting rid of storing the interface name in the endpoint? Isn't that going to break things for some people?